thunder/torch/__init__.py

```diff
@@ -599,6 +599,46 @@ def zeros_like(a: TensorLike, /, *, device: DeviceLike | None = None, dtype: dty
     return full_like(a, 0, device=device, dtype=dtype)
 
 
+@torchsymbol(torch.empty)
+def empty(
+    *size: int,
+    out: None | TensorLike = None,
```

Comment:
Nit, let's move the `device` and `dtype` parameters to be after `size` since they're the two parameters that can be set and are common to use